parsers/s01-parse/eastcw/foundryvtt-logs.yaml

```diff
@@ -0,0 +1,33 @@
+name: eastcw/foundryvtt-logs
+description: "Parse Foundry VTT logs"
+filter: "evt.Parsed.program == 'foundryvtt'"
+debug: true
+onsuccess: next_stage
+pattern_syntax:
+  DATE_YMD: "%{YEAR:year}-%{MONTHNUM:month}-%{MONTHDAY:day}"
+
+nodes:
```

Comment:
For the grok parsing since its all JSON maybe we should use the inbuilt https://docs.crowdsec.net/docs/next/expr/json_helpers#unmarshaljsonjsonblob-string-out-mapstringinterface-targetkey-string

Then it doesnt matter if the json format changes since we parse the keys to an object string

Reply:
Good note. Is there a way to test a setup that uses UnmarshalJSON() ? Trying to use hubtest on any yaml that calls it throws a fatal error, even parsers that I know should be valid like crowdsecurity/laurel-logs